networking/src/main/java/com/lyft/networking/apiObjects/internal/ICompleteData.java

```diff
@@ -0,0 +1,21 @@
+package com.lyft.networking.apiObjects.internal;
+
+/**
+ * INTERNAL USE ONLY.
+ *
+ * An interface used to verify that Response objects have been rendered correctly and are valid to serve to the client.
+ */
+public interface ICompleteData {
```

Comment:
I would rather use a clear adjective here like `Validatable`. I don't like `I` in interfaces name especially when users can see them.